package/gluon-core/check_site.lua

```diff
@@ -28,6 +28,14 @@ for _, config in ipairs({'wifi24', 'wifi5'}) do
     need_string('regdom') -- regdom is only required when wifi24 or wifi5 is configured
 
     need_number(config .. '.channel')
+
+    local rates = {1000, 2000, 5500, 6000, 9000, 11000, 12000, 18000, 24000, 36000, 48000, 54000}
+    local supported_rates = need_array_of(config .. '.supported_rates', rates, false)
+    if supported_rates then
```

Comment:
@NeoRaider: I don't like this `if`, but I think it is more clear than:
`need_array_of(config .. '.basic_rate', supported_rates or rates, supported_rates and true or false)`